pkg/skaffold/initializer/init.go

```diff
@@ -288,9 +288,12 @@ func detectBuilders(enableJibInit, enableBuildpackInit bool, path string) ([]Ini
 	}
 
 	// Check for Dockerfile
-	if docker.Validate(path) {
-		results := []InitBuilder{docker.ArtifactConfig{File: path}}
-		return results, true
+	base := filepath.Base(path)
+	if strings.Contains(base, "Dockerfile") {
+		if docker.Validate(path) {
```

Comment:
We could also check in `docker.Validate(path)` if there is some python-ish code after `from ...` (like `import <name>`). 

WDYT @dgageot

Reply:
I don't think that Dockerfile validator should be aware of python-ish code constructs.
It's responsibility to validate Dockerfile and it does that by validating commands. Validating arguments would be too much overhead IMO.